augly/video/transforms.py

```diff
@@ -2074,9 +2140,9 @@ def apply_transform(
 
 class VStack(BaseTransform):
     def __init__(
-        self,
-        second_video_path: str,
-        use_second_audio: bool = False,
+        self, 
+        second_video_path: str, 
+        use_second_audio: bool = False, 
```

Comment:
why have these spaces been added? please remove
```suggestion
        self,
        second_video_path: str,
        use_second_audio: bool = False,
```